src/command/utils.rs

```diff
@@ -37,6 +37,7 @@ fn find_manifest_from_cwd() -> Result<PathBuf, failure::Error> {
 
 /// Construct our `pkg` directory in the crate.
 pub fn create_pkg_dir(out_dir: &Path) -> Result<(), failure::Error> {
+    let _ = fs::remove_dir_all(&out_dir); // Clean up any existing directory and ignore errors
```

Comment:
Hmm. It's unclear if we need to remove the entire out directory. It may not be a big deal but say someone has some `npm link` going on pointed at the `pkg` directory, this may break it on every rebuild. I'm not sure if `npm link` does a symlink or a hard link (or is that not a thing with directories) which I think would be the difference.

Reply:
Yeah, I can see your point. `npm link` [documentation claims they are symlinks](https://docs.npmjs.com/cli/v7/commands/npm-link).

That said, I don't feel strongly about this. I personally tend to want my tests to run on a as clean as possible environment to avoid spurious failures due to other tests leaving cruft behind, but I'm happy to remove this cleanup if that makes sense to you.

Reply:
We're seeing errors that I think are in relation to this as reported at https://github.com/rustwasm/wasm-pack/issues/1099#issuecomment-1033430935

Reply:
I've raised a PR to revert this change: https://github.com/rustwasm/wasm-pack/pull/1110